src/pyrokinetics/kinetics/kinetics.py

```diff
@@ -1,5 +1,7 @@
 from typing import List, Optional
 
+import matplotlib.pyplot as plt
```

Comment:
`matplotlib` is imported within the function that uses it, so we don't need it here. Normally I'd prefer to keep all imports at the top of the file, but I recall running into issues earlier with Pyrokinetics taking several seconds to import, and Matplotlib was one of the worst culprits.